flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/internal/KafkaConsumerThread.java

```diff
@@ -482,6 +511,67 @@ void reassignPartitions(List<KafkaTopicPartitionState<TopicPartition>> newPartit
 		return new KafkaConsumer<>(kafkaProperties);
 	}
 
+	@VisibleForTesting
+	RateLimiter getRateLimiter() {
+		return rateLimiter;
+	}
+
+	// -----------------------------------------------------------------------
+	// Rate limiting methods
+	// -----------------------------------------------------------------------
+
+	/**
+	 * Set the {{@link #bytesPerSecondMax}} per consumer using the global bytes per second.
+	 */
+	public void setBytesPerSecondMax() {
```

Comment:
Is this intended to be public or is that a mistake?  It looks like this should be private.

Reply:
Not intended. Fixed.